### ISSUE TITLE
Doesn't work with MagicPython (weird)

### ISSUE THREAD

**Issue by danqing** [Glavin001/atom-beautify#1851]
# Description

So I use MagicPython instead of language-python with Atom. (Many people do.) Beautify works fine with .py files (even though MagicPython shows in the bottom bar). The problem is when I create a new file and set MagicPython as its syntax. If I try to then beautify, I get the following warning:

Atom Beautify could not find a supported beautifier for this file
Atom Beautify could not determine a supported beautifier to handle this file with grammar "MagicPython" and extension "undefined". If you would like to request support for this file and its language, please create an issue for Atom Beautify at https://github.com/Glavin001/atom-beautify/issues

# Steps to Reproduce

1. Create new file and set MagicPython as the syntax (need to have MagicPython installed)
2. Run command `Atom Beautify: Beautify Editor`
3. Warning appears

In short, can we have MagicPython behave exactly the same as Python?

Thanks! This is one of the most important plugins :)

**Comment by danqing** [comment]
Just to clarify, while it seems that if I save the new file as something.py that would make the trick, it's important to be able to beautify without saving the file. I use that on a daily basis - open atom, throw some python code in *just* to use beautify, and then i will send it out / use it elsewhere. It'd be great if I don't need to create a file everytime I do it.

**Comment by danqing** [comment]
Still an issue.

**Comment by szeck87** [comment]
Let me look into it, it _looks_ like a fairly simple change.

**Comment by szeck87** [comment]
@danqing Can you confirm this is the Atom package you are using for MagicPython?  Want to make sure I test it out properly:  https://atom.io/packages/magicpython

**Comment by danqing** [comment]
Yes!

**Comment by Glavin001** [comment]
Published to v0.32.0
